External/Plugins/CodeRefactor/Provider/ICommandFactory.cs

```diff
@@ -122,23 +122,23 @@ public interface ICommandFactory
         /// Outputs found results.
         /// Uses the current text location as the declaration target.
         /// </summary>
-        Command CreateRenameCommand();
+        Command CreateRenameCommandAndExecute();
```

Comment:
This change feels strange. Why AndExecute? Is the command executed by itself? At least in the Factory itself is not run. I would say it's not right if this command behaves in a way all the other commands in a different one.